### ISSUE TITLE
DeprecationWarning: 'updateFunctionDeclaration' has been deprecated since v4.8.0. 

### ISSUE THREAD

**Issue by hungtcs** [Jack-Works/react-refresh-transformer#18]
```
DeprecationWarning: 'updateFunctionDeclaration' has been deprecated since v4.8.0. 
Decorators are no longer supported for this function. Callers should switch to an overload that does not accept a 'decorators' parameter.
```

**Comment by Jack-Works** [comment]
hi! I have published 2.0.8, please try it

**Comment by hungtcs** [comment]
> hi! I have published 2.0.8, please try it

🎉 Thank you, 2.0.8 is working for me.
